fcl/dunefd/reco/workflow_reco_dune10kt.fcl

```diff
@@ -70,7 +73,8 @@ dunefd_horizdrift_producers:
   energyrecnue:          @local::dunefd_nuenergyreco_pandora_nue
   energyrecnc:           @local::dunefd_nuenergyreco_pandora_nc
   # photon detector reconstruction
-  ophit:      @local::dunefd_ophit
+  opdec:      @local::dune_deconvolution
+  ophitspe:   @local::dune_ophit_finder_deco
```

Comment:
The producer labels are quite vague e.g. 'dune_deconvolution' rather than 'dunefd_deconvolution'.  I wonder if we're at risk of another user for another detector making changes to these upstream and we get unintentionally impacted

Reply:
Good catch, Dom. We should include dunefd in the name in the future. I think Maritza is working on the deconvolution for pdhd and it could be done when she is ready for a PR so that we have a dunefd_deconvolution and protodunehd_deconvolution, the same for ophit_finder_deco.